views/home.hbs

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
```

Comment:
nit: the indent is inconsistent in this file: it starts with 4 spaces here but later uses 2 spaces in the body.

Reply:
Updated so the entire doc is indented consistently.